### PR TITLE
Parse URL query params, and add it to the list of parameters that are send to the controllers or URLOpenHandler

### PR DESCRIPTION
**Pull request by juancruzmdq**
ok, Now are in a diferent PR

### CODE REVIEW

**Comment by devxoul** [comment]
`values` should not contain query parameters because:

1. It is originally designated to provide URL placeholder values.
2. Query parameters will replace URL placeholder values if they have same name.

**Comment by juancruzmdq** [comment]
ok, but could be useful have access to all parameters. in my project I will need

**Comment by devxoul** [comment]
For example: https://github.com/devxoul/URLNavigator#where-to-map-urls

```
Navigator.map("myapp://alert") { URL, values in
    print(URL.parameters["title"])
    print(URL.parameters["message"])
    self.someUtilityMethod()
    return true
}
```

**Comment by juancruzmdq** [comment]
ok, I was waiting this values in the "values" parameter.
Do you have some kind of extension to access URL.parameters["..."] ?

File: Sources/URLNavigator.swift [hunk 1]
```diff
@@ -151,6 +151,13 @@ public class URLNavigator {
                 }
             }
 
+            
+            // Store any parameter from the URL's query
+            let urlComponents = NSURLComponents(string: URL.URLStringValue)
+            urlComponents?.queryItems?.forEach({ (param) in
```
Comment (devxoul) [inline Sources/URLNavigator.swift]: `queryItems` is only available on iOS 8+

**Comment by devxoul** [comment]
@juancruzmdq, you can write your own extension, or it'll be great to URLNavigator to provide it.

**Comment by juancruzmdq** [comment]
yes, don't worry about this PR, you can close it

**Comment by devxoul** [comment]
@juancruzmdq, I've made a new PR: #13. If it fits your needs, I'll merge it and make a new release including #11 :)

**Pull request closed without merging**